web/src/invite.js

```diff
@@ -204,7 +210,13 @@ function valid_to(expires_in) {
     return $t({defaultMessage: "Expires on {date} at {time}"}, {date, time});
 }
 
-function get_expiration_time_in_minutes() {
+export function get_expiration_time_in_minutes(
+    custom_expiration_time_unit,
+    custom_expiration_time_input,
+) {
+    if ($("#expires_in").val() !== "custom") {
```

Comment:
We probably want to pass in `$("#expires_in")` to make the caller be explicit about the element it's acting on.